projects/plugins/jetpack/modules/comments/subscription-modal-on-comment/class-jetpack-subscription-modal-on-comment.php

```diff
@@ -49,11 +47,14 @@ public static function get_block_template_part_id() {
 	 * Limited to Atomic sites.
 	 */
 	public function __construct() {
-		if ( ( new Host() )->is_woa_site() && get_option( 'jetpack_verbum_subscription_modal', true ) ) {
-			add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_assets' ) );
-			add_action( 'wp_footer', array( $this, 'add_subscription_modal_to_frontend' ) );
+		// Only Automattitians for now
+		if ( defined( 'AT_PROXIED_REQUEST' ) && AT_PROXIED_REQUEST ) {
+			if ( ( new Host() )->is_woa_site() && get_option( 'jetpack_verbum_subscription_modal', true ) ) {
+				add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_assets' ) );
+				add_action( 'wp_footer', array( $this, 'add_subscription_modal_to_frontend' ) );
+			}
+			add_filter( 'get_block_template', array( $this, 'get_block_template_filter' ), 10, 3 );
```

Comment:
I think we can get rid of this since we don't want to show anything if the feature is not enabled or is a self-hosted site.